Assets/HoloToolkit/Input/Scripts/Focus/FocusManager.cs

```diff
@@ -552,8 +552,8 @@ private void RaycastPhysics(PointerData pointer, LayerMask[] prioritizedLayerMas
             RayStep rayStep = default(RayStep);
             RaycastHit physicsHit = default(RaycastHit);
 
-            Debug.Assert(pointer.PointingSource.Rays != null, "No valid rays for " + pointer.GetType());
```

Comment:
Doesn't this remove context from the assertion statement? Why would we want to remove that?

Reply:
`GetType()` is causing an enormous amount of allocations during this time.

Reply:
Do you have a measured performance increase?